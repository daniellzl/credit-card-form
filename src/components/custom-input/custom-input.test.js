import { shallow } from "enzyme";
import CustomInput from "./custom-input.component.jsx";

console.log(shallow(<CustomInput />).debug());

it("expect to render CustomInput component", () => {
  expect(shallow(<CustomInput />)).toMatchSnapshot();
});
