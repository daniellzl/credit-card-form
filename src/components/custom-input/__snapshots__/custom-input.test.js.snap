// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CustomInput component 1`] = `
<div
  className="customInputFrame"
>
  <input
    className="customInput"
  />
  <label
    className="customInputLabel  "
  />
</div>
`;
